dist/src/main/java/io/camunda/application/commons/search/ElasticSearchSearchClientConfiguration.java

```diff
@@ -31,4 +37,30 @@ public ElasticsearchSearchClient elasticsearchSearchClient(
     final var elasticsearch = connector.createClient();
     return new ElasticsearchSearchClient(elasticsearch);
   }
+
+  @Bean(destroyMethod = "close")
+  public RestHighLevelClient restHighLevelClient(final SearchClientProperties configuration) {
```

Comment:
we are actually trying to get rid of ElasticsearchClient 7 dependency (which brings the `RestHighLevelClient`), is it possible to use `co.elastic.clients.elasticsearch.ElasticsearchClient`?